### DISCUSSION TITLE
no captcha in any sites with cloudflare

### DISCUSSION THREAD

**Discussion by f1ns1ck** [Atlas-OS/Atlas#1316] [Help & Support] [answered]
.

**Comment by Ast3risk-ops** [comment]
@f1ns1ck please use English for discussions and issues.

**Comment by f1ns1ck** [comment]
no captcha in any sites with cloudflare

**Reply by Ast3risk-ops** [reply, marked as answer]
Check that your system clock is set properly and that `challenges.cloudflare.com` is accessible. Not an Atlas issue.
